Tensile/KernelWriterAssembly.py

```diff
@@ -2294,8 +2295,9 @@ def defineFlatMemoryMacros(self):
     for t in type_list:
       origin  = f'{t}'
       replace = f'{t}' if self.archCaps["InstRename"] else f'{type_list[t]}'
-      kStr += self.generalMacro('flat_load_', origin, replace, 'dst', 'base', 'md0', 'md1', 'md2') + self.endLine
-      kStr += self.generalMacro('flat_store_', origin, replace, 'base', 'src', 'md0', 'md1', 'md2') + self.endLine
+      # use global_load/store instead of flat instructions
+      kStr += self.generalMacro('global_load_', origin, replace, 'dst', 'base', 'md0', 'md1', 'md2') + self.endLine
+      kStr += self.generalMacro('global_store_', origin, replace, 'base', 'src', 'md0', 'md1', 'md2') + self.endLine
 
     type_list = {'_b32': ''}
     for t in type_list:
```

Comment:
Should we also switch out the flat_atomic_cmpswap on line 2306 for global_atomic_cmpswap?

Reply:
If we're updating here, it also shows up on line 11859 and 11989

Reply:
Thanks. Update.

Reply:
Last time, I did not verify global_atomic_cmpswap, but this time, I verified it.